meta/main.yml

```diff
@@ -1,10 +1,10 @@
 ---
 galaxy_info:
-  author: Markus Langer
```

Comment:
btw, this was needed because otherwise ansible cannot compute the FQCN. we can also leave the author as-is and add a `namespace` instead

Reply:
okay 👍